tree/dataframe/inc/ROOT/RActionSnapshot.hxx

```diff
@@ -0,0 +1,257 @@
+#ifndef ROOT_RACTIONSNAPSHOT
+#define ROOT_RACTIONSNAPSHOT
+
+#include "ROOT/RDFNodes.hxx"
+#include "ROOT/RDFActionHelpers.hxx"
+#include "TError.h"
```

Comment:
Are we using something which is inside this header?

Reply:
thanks, fixed